benchmark/src/main/java/com/amazon/opendistroforelasticsearch/sql/benchmark/utils/load/mysql/MysqlDataLoader.java

```diff
@@ -15,14 +15,34 @@
 
 package com.amazon.opendistroforelasticsearch.sql.benchmark.utils.load.mysql;
 
+import com.amazon.opendistroforelasticsearch.sql.benchmark.BenchmarkService;
+import com.amazon.opendistroforelasticsearch.sql.benchmark.utils.BenchmarkConstants;
 import com.amazon.opendistroforelasticsearch.sql.benchmark.utils.load.DataFormat;
 import com.amazon.opendistroforelasticsearch.sql.benchmark.utils.load.DataLoader;
+import com.datastax.driver.core.Cluster;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.Statement;
+import java.util.LinkedList;
+import java.util.Map;
 
 /**
  * Data loader for MySQL database.
  */
 public class MysqlDataLoader implements DataLoader {
 
+  private static String url = "jdbc:mysql://localhost/";
+  private Connection connection = null;
+  private Statement statement = null;
+
+  /**
+   * Constructor for MysqlDataLoader.
+   */
+  public MysqlDataLoader() {
+    url +=
```

Comment:
Modification of statics is not a good idea. If you create two MysqlDataLoader's then you've messed up your URL. Please change to have a static final initial portion of the URL that then creates a local variable for the URL.

Reply:
added local var